fbw-common/src/systems/shared/src/navigraph/client.ts

```diff
@@ -1,348 +1,33 @@
-              }
-            : undefined,
-        }));
-
-        return {
-          arrival: chartArray.filter((chart) => chart.type.category === 'ARRIVAL'),
-          approach: chartArray.filter((chart) => chart.type.category === 'APPROACH'),
-          airport: chartArray.filter((chart) => chart.type.category === 'AIRPORT'),
-          departure: chartArray.filter((chart) => chart.type.category === 'DEPARTURE'),
-          reference: chartArray.filter(
-            (chart) =>
-              chart.type.category !== 'ARRIVAL' &&
-              chart.type.category !== 'APPROACH' &&
-              chart.type.category !== 'AIRPORT' &&
-              chart.type.category !== 'DEPARTURE',
-          ),
-        };
-      }
-    }
-
-    return emptyNavigraphCharts;
-  }
-
-  public async getAirportInfo(icao: string): Promise<AirportInfo | null> {
-    if (this.hasToken) {
-      const chartJsonUrl = await this.chartCall(icao, 'airport.json');
-
-      const chartJsonResp = await fetch(chartJsonUrl);
-
-      if (chartJsonResp.ok) {
-        const chartJson = await chartJsonResp.json();
-
-        return { name: chartJson.name };
-      }
-    }
-
-    return null;
-  }
-
-  public get hasToken(): boolean {
-    return !!this.accessToken;
-  }
-
-  public async assignUserName(): Promise<void> {
-    if (this.hasToken) {
-      try {
-        const userInfoResp = await fetch('https://identity.api.navigraph.com/connect/userinfo', {
-          headers: { Authorization: `Bearer ${this.accessToken}` },
-        });
-
-        if (userInfoResp.ok) {
-          const userInfoJson = await userInfoResp.json();
-
-          this.userName = userInfoJson.preferred_username;
-          NXDataStore.set('NAVIGRAPH_USERNAME', this.userName);
-        }
-      } catch (_) {
-        console.log('Unable to Fetch User Info. #NV103');
-      }
-    }
-  }
-
-  public async fetchSubscriptionStatus(): Promise<NavigraphSubscriptionStatus> {
-    if (this.hasToken) {
-      const decodedToken = JSON.parse(atob(this.accessToken.split('.')[1]));
-
-      const subscriptionTypes = decodedToken.subscriptions as string[];
-
-      if (subscriptionTypes.includes('fmsdata') && subscriptionTypes.includes('charts')) {
-        return NavigraphSubscriptionStatus.Unlimited;
-      }
-    }
-
-    return NavigraphSubscriptionStatus.None;
-  }
+    return !(NavigraphKeys.clientSecret === '' || NavigraphKeys.clientId === '');
+  }
+
+  // public async amdbCall(query: string): Promise<string> {
+  //   const callResp = await fetch(`https://amdb.api.navigraph.com/v1/${query}`, {
+  //     headers: {
+  //       Authorization: `Bearer ${this.accessToken}`,
+  //     },
+  //   });
+  //
+  //   if (callResp.ok) {
+  //     return callResp.text();
+  //   }
+  //
+  //   // Unauthorized
+  //   if (callResp.status === 401) {
+  //     await this.getToken();
+  //
+  //     return this.amdbCall(query);
+  //   }
+  //
+  //   return Promise.reject();
+  // }
```

Comment:
```suggestion
```
This is likely to get left behind when the OANS PR is rebased as it might not conflict or be detected as the same by git with the //.

Reply:
Fixed.